internal/pipeline/merging.go

```diff
@@ -119,13 +121,15 @@ func (m *filesystemMerging) HandleCancel(cancel incoming.CancelACHFile) error {
 }
 
 func (m *filesystemMerging) isolateMergableDir() (string, error) {
-	// rename m.baseDir so we're the only accessor for it, then recreate m.baseDir
-	parent, _ := filepath.Split(m.baseDir)
-	newdir := filepath.Join(parent, time.Now().Format("20060102-150405"))
-	if err := os.Rename(m.baseDir, newdir); err != nil {
+	// rename the shard directory so we're the only accessor for it, then recreate it
```

Comment:
where do we recreate the directory?

Reply:
it seems we create the directory if it does not exist when we handle the received file. am I right?

Reply:
In the same location it was previously? This is isolating `storage/mergable/` and recreating it.

Reply:
I see that previously it was `return newdir, os.Mkdir(m.baseDir, 0777)` - we were re-creataing dir on the return, while now I don't see it.

but I see we have `os.MkdirAll(path, 0777)` inside `writeACHFile`.

Reply:
Yea. I had to make [this change as well](https://github.com/moov-io/achgateway/commit/9312072424c975e11005d7608de734c05b53150a) so directories are created as needed.